scripts/run_design.py

```diff
@@ -90,8 +92,15 @@ def main():
     for arg_name, enum in flow_args_map.items():
         flow_args[enum] = getattr(args, arg_name)
 
-    run_design(Path(args.design_path), args.flow, args.error_flow, flow_args=flow_args)
+    # Convert the design path to an absolute path
+    design_path = Path(os.path.expanduser(args.design_path))
+
+    # Set experiment_dir to the current working directory if not provided
+    experiment_dir = Path.cwd() if args.experiment_dir is None else Path(os.path.expanduser(args.experiment_dir))
```

Comment:
Again, I don't think you need to `expanduser` as that should have already been done by the shell?

Reply:
reverted